### ISSUE TITLE
Add glassmorphism css

### ISSUE THREAD

**Issue by rhnas110** [sanskarmalkhede/calculator#17]
Issue Description:
I want to add glassmorphism css, can you please have a quick look to see if it looks nice or not? and if @sanskarmalkhede  thinks it's nice, can you please assign this task to me?

Before:
![image](https://github.com/sanskarmalkhede/calculator/assets/51848196/737af4c3-56e1-402a-b212-3e4b2a5bf9e2)

After:
![image](https://github.com/sanskarmalkhede/calculator/assets/51848196/17bfc25f-e338-46bb-9a4d-c789d4b3beaf)

and @sanskarmalkhede  could you add this issue with hacktoberfest tag? Thanks Sir

**Comment by sanskarmalkhede** [comment]
I don't think it will make a huge difference. If you have more suggestions on background, glassmorphism css, buttons, text and it's colors please let me know.
